### DISCUSSION TITLE
Students not receiving their PW reset emails

### DISCUSSION THREAD

**Discussion by drogrady** [community/community#35329] [GitHub Education]
I have a group of 33 students and some have asked for PW resets. However their email domain (not in my control) is blocking the reset email. 
Is there a way for us to manually change their passwords?

I put in a ticket nearly a week ago. It's unfortunate that Github has been nonresponsive.
I have another 80 students I would like to start with Github, but it's not going to happen if we can't get this situation resolved. 
This is NOT the way to run an educational technology website.

Extremely disappointing

**Comment by airtower-luna** [comment]
Have you sent a bug report to the email admins yet? Spam blocking is nice and well, but they must have a way to prevent overblocking like this.
